Sources/SwiftSyntax/generated/SyntaxTraits.swift

```diff
@@ -72,6 +72,22 @@ public protocol DeclGroupSyntax: SyntaxProtocol, DeclSyntaxProtocol {
     set
   }
   
```

Comment:
Also open to renaming `DeclGroupSyntax` if anyone has a better name for it.

Reply:
I’m not a huge fan of my suggestion but what this conceptually is, is a `DeclWithMembersSyntax`, I think. At least that describes the trait IMO.

Reply:
That matches the other "With" at least 😅